eng/pipelines/typespec-apiview.yml

```diff
@@ -0,0 +1,59 @@
+parameters:
+  - name: APIViewArtifactsDirectoryName
+    type: string
+    default: 'TypespecAPIViewArtifacts'
+  - name: APIViewArtifactsName
+    type: string
+    default: 'typeSpecAPIViewArtifacts'
+  - name: APIViewAPIUri
+    type: string
+    default: 'https://apiview.dev/PullRequest/DetectAPIChanges'
+    # Please use 'https://apiviewstagingtest.com/PullRequest/DetectAPIChanges' for testing purposes
+
+pr:
+   branches:
+     include:
+       - main
```

Comment:
I know this matches the swagger-apiview.yml file but we will likely want to open this up to run on all branches not just main.